socialrobot_actionlib/Action action
socialrobot_actionlib/Predicate[] current_states
---
bool result
